src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,53 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    public static int[] getReportFromFile(String fileName) {
+        File file = new File(fileName);
+        try (BufferedReader reader = new BufferedReader(new FileReader(file))) {
+            String line = reader.readLine();
+            int[] report = new int[2];
+
+            while (line != null) {
+                String[] splitLine = line.split(",");
+                if (splitLine[0].equals("supply")) {
```

Comment:
If you have strange strings or numbers in the code, it's better to declare them as constants. The name of the constant should display this object's purpose.